src/js/polyfills.js

```diff
@@ -196,3 +196,22 @@ if (typeof Object.assign !== 'function') {
     configurable: true,
   });
 }
+
+// from: https://developer.mozilla.org/en-US/docs/Web/API/Element/closest#Polyfill
+if (!Element.prototype.matches) {
+  Element.prototype.matches =
+    Element.prototype.msMatchesSelector ||
+    Element.prototype.webkitMatchesSelector;
+}
+
+if (!Element.prototype.closest) {
+  Element.prototype.closest = function (s) {
+    var el = this;
+
+    do {
+      if (Element.prototype.matches.call(el, s)) return el;
+      el = el.parentElement || el.parentNode;
+    } while (el !== null && el.nodeType === 1);
+    return null;
+  };
+}
```

Comment:
Are you use Babel to transpile the JS? If so, and you have included IE11 in the support matric it might already be doing this?